packages/react/src/ComponentStyles.stories.tsx

```diff
@@ -0,0 +1,48 @@
+import * as React from 'react';
```

Comment:
As stories does not have 1:1 mapping to files, can we move them to `/stories` subdirectory? I.e. `packages/react/src/ComponentStyles.stories.tsx` => `packages/react/src/stories/ComponentStyles.stories.tsx`